lib/tasks/compare.rake

```diff
@@ -3,6 +3,8 @@ namespace :comparison do
   desc "report on differences in items tagged as reported by rummager and contentapi"
   task :run => :environment do
     require 'csv'
+    require 'gds_api/content_api'
+    CollectionsPublisher.services(:content_api, GdsApi::ContentApi.new(Plek.new.find('content_api')))
```

Comment:
I think that'll need to be `Plek.new.find('contentapi')` (without the underscore...).  The argument to Plek becomes the hostname in the returned URL.